offline-runtime/data/src/com/intellij/rt/coverage/offline/RawHitsReport.java

```diff
@@ -33,40 +30,24 @@
 public class RawHitsReport {
   private static final int MAGIC = 284996684;
 
-  private static boolean reportFileExists = "true".equals(System.getProperty("kover.offline.report.exists", "false"));
+  public static void dump(DataOutput output, RawProjectData data) throws IOException {
```

Comment:
What is the purpose of dumping into a `DataOutput` (not `File`), if the report can be parsed only by using `load(File)` method?

Reply:
But it must be saved into a file later anyway

Reply:
How and when to do this will be decided by the Kover API user